### PR TITLE
updates-for-tags-and-defenders

### PR DESCRIPTION
**Pull request by ssugandh**
For PCSUP-7947: Added another example for tags-cve-vuln api to tag a CVE to all packages and resources.

Corrected cURL examples for Defenders /daemonset_yaml and helm.

### CODE REVIEW

File: api/descriptions/tags/tag_cve_post.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 Sets a tag to a vulnerability based on Common Vulnerability and Exposures (CVE) ID, package, and resource.
-When you apply a tag to a vulnerability only on a package, the tag applies to the vulnerability in all the resources related to it.
+
```
Comment (ssugandh) [inline api/descriptions/tags/tag_cve_post.md:2]: @MorShabiPAN : Can you please review the note?

File: api/descriptions/tags/tag_cve_post.md [hunk 2]
```diff
@@ -69,3 +72,29 @@ Consider the following scenarios for source and binary packages:
   Refer to the following image that displays the tagged vulnerability:
   
   ![Tagged vulnerability in CentOS](https://cdn.twistlock.com/docs/api/CentOS-Vuln-Bin-Package-CVE-tagged-Ignored-Vuln.png)
+
```
Comment (ssugandh) [inline api/descriptions/tags/tag_cve_post.md:76]: @MorShabiPAN : Can you please review the example?

File: api/descriptions/tags/tag_cve_post.md (hunk 1 above)
Comment (MorShabiPAN) [reply api/descriptions/tags/tag_cve_post.md:2]: looks good

File: api/descriptions/tags/tag_cve_post.md (hunk 2 above)
Comment (MorShabiPAN) [reply api/descriptions/tags/tag_cve_post.md:76]: looks good

File: api/descriptions/tags/tag_cve_post.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 Sets a tag to a vulnerability based on Common Vulnerability and Exposures (CVE) ID, package, and resource.
-When you apply a tag to a vulnerability only on a package, the tag applies to the vulnerability in all the resources related to it.
 
+**Consider the following scenarios**: 
+  - When you apply a tag to a vulnerability only on a package, the tag applies to the vulnerability in all the resources related to it.
+  - When you apply a tag only to a vulnerability, the tag applies to the vulnerability in all the packages and resources related to it.
+  - When you apply a tag to a vulnerability on a resource type, specify the scope of the resources using either a wildcard "*" or resource names.
```
Comment (ssugandh) [inline api/descriptions/tags/tag_cve_post.md:6]: @MorShabiPAN : Added this point to explain the relationship for resource and resource types. Followed by examples below. Please review.

File: api/descriptions/tags/tag_cve_post.md [hunk 4]
```diff
@@ -69,3 +73,70 @@ Consider the following scenarios for source and binary packages:
...
+    "id": "CVE-2020-16156",
+    "packageName": "*"
+  }' \
+    "https://<CONSOLE>/api/v<VERSION>/tags/Ignored/vuln"
+  ```
+  Refer to the following image that displays the tagged vulnerability:
+  
+  ![Tagged vulnerability](https://cdn.twistlock.com/docs/api/Tagged-Vulnerability.png)
+
+- A CVE `CVE-2020-16156` is found in several packages such as `perl`, `perl-open`, `perl-macros`, `perl-libs`, and so on. You want to apply a tag `Ignored` to the resource type `image` but to all the packages and resources.
```
Comment (ssugandh) [inline api/descriptions/tags/tag_cve_post.md:103]: @MorShabiPAN : New example added as discussed over email. Please review.

File: api/descriptions/tags/tag_cve_post.md (hunk 3 above)
Comment (MorShabiPAN) [reply api/descriptions/tags/tag_cve_post.md:6]: looks good

**Review by iansk — APPROVED** [review]

**Comment by iansk** [comment]
@ssugandh Looks good. This is merged.

**Pull request merged**